diff --git a/__init__.py b/__init__.py
index 3ffd16f..bf9ee03 100644
--- a/__init__.py
+++ b/__init__.py
@@ -15,7 +15,6 @@ _package_data = dict(
     entry_points=None,
     since=2014,
     extras_require={
-        ':platform_python_implementation=="CPython"': ['ruamel.yaml.clibz>=0.3.3'],  # NOQA
         'jinja2': ['ruamel.yaml.jinja2>=0.2'],
         'docs': ['ryd', 'mercurial>5.7'],
     },
